styles.css

```diff
@@ -0,0 +1,481 @@
+  }
+
+  .logo {
+    padding: 16% 1% 0% 190%;
+    height: 60%;
+    width: 97%;
+  }
+
+  .ham {
+    display: none;
+  }
+
+  .menu {
+    display: flex;
+    backdrop-filter: none;
+    background: #ffff;
+    height: 0;
+    position: static;
+  }
+
+  .nav-content {
+    display: flex;
+    flex-direction: row;
+    font-family: 'Lato', sans-serif;
+    font-size: 17px;
+    color: #00000094;
+    padding: 1.5% 0 2% 1%;
+    gap: 9%;
+  }
+
+  .nav-content li {
+    text-decoration: none;
+    color: #00000094;
+  }
+
+  #cc-link {
+    display: flex;
+    border: 5px solid orangered;
+    align-items: center;
+    justify-content: center;
+    font-size: 17px;
+    font-weight: 700;
+    margin: -8% 1% 1% 1%;
+    height: 2vh;
+    width: 10vw;
+    padding: 5% 3% 5% 3%;
+  }
+
+  .close-button {
+    display: none;
+  }
+
+  #nav-grey {
+    color: #00000094;
+  }
+
+  #hello {
+    font-size: 42px;
+  }
+
+  .title-intro h1 {
+    font-size: 50px;
+    width: 55vw;
+  }
+
+  .summit-intro {
+    font-size: 25px;
+    width: 60vw;
+  }
+
+  .date .show-date {
+    font-size: 30px;
+  }
+
+  .date p {
+    font-size: 20px;
+  }
+
+  .grid-program {
+    display: grid;
+    grid-template-rows: 0.4fr 0.1fr 0.3fr;
+    grid-template-columns: none;
+    height: 28vh;
+    margin: 0.2%;
+    padding: 1% 1% 1% 1%;
+    width: 12%;
+  }
+
+  .grid-program:hover {
+    background-color: #00000045;
+  }
+
+  #gr-program {
+    display: flex;
+    flex-direction: row;
+    text-align: center;
+    justify-content: center;
+  }
+
+  .program-icon {
+    margin-left: 20%;
+  }
+
+  .program-points {
+    font-weight: 700;
+  }
+
+  .program-ex {
+    font-size: 94%;
+  }
+
+  .sm-line {
+    margin-left: 49%;
+    margin-top: -7%;
+  }
+
+  .btn-summit {
+    display: none;
+  }
+
+  #see-program1 {
+    display: block;
+    font-size: 20px;
+    text-decoration: underline;
+    color: #ffff;
+    font-family: 'Lato', sans-serif;
+    padding: 7% 1% 1% 2%;
+    text-align: center;
+  }
+
+  .feature {
+    padding: 5% 1% 4% 3%;
+  }
+
+  .feature-dash {
+    width: 5vw;
+    margin: 0% 1% 1% 48%;
+  }
+
+  #ftr-speakers {
+    background-color: #0000000d;
+  }
+
+  .summit-speaker {
+    margin: 1% 15% 1% 12%;
+  }
+
+  #speaker-card-container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    grid-column-gap: 5px;
+    grid-row-gap: 5px;
+  }
+
+  .line-speaker {
+    width: 3vw;
+  }
+
+  .image-speaker {
+    /* width: 12vw;
+    height: 70%; */
+    width: 9vw;
+    height: 60%;
+  }
+
+  .attendees-info {
+    padding: 1% 9% 1% 9%;
+  }
+
+  .partners {
+    display: block;
+    background-color: hsl(0deg 0% 13% / 97%);
+    padding: 1% 7% 1% 7%;
+  }
+
+  .ab-prts {
+    color: #ffff;
+    font-size: 25px;
+    padding: 1% 2% 1% 1%;
+    font-family: 'Lato', sans-serif;
+    text-align: center;
+  }
+
+  .about-partners {
+    /* width: 100%;
+    padding: 3% 2% 1% 2%; */
```

Comment:
- [ ] - [ ] [OPTIONAL] Also, it could be better to remove this line comments for good practice, 👍

Reply:
Fixed